package.json

```diff
@@ -46,8 +46,8 @@
     "shallow-equal": "^1.1.0"
   },
   "peerDependencies": {
-    "@types/react": "^16.0.0",
-    "react": "^16.3.0"
+    "@types/react": ">= 16.3.0",

```

Comment:
yeah this makes us allow `17.0.0` so might be dangerous

Reply:
Agreed we should only be using ^16.0.0